test/browser_tests/conf.js

```diff
@@ -2,27 +2,80 @@
 
 var env = require( './environment.js' );
 
-exports.config = {
+// Set browsers to test.
+// See https://docs.saucelabs.com/reference/platforms-configurator/
+// for configuration values for version and platform.
+
+// Essential browsers for running locally.
+var _essential = {
   multiCapabilities: [
     {
       browserName: 'chrome',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    {
-      browserName: 'firefox',
+      version:     '',
+      platform:    'WINDOWS',
+      maxDuration: 10800,
       specs:       [ 'spec_suites/shared/*.js' ]
     },
     // Large Screen only tests
     {
       browserName:   'chrome',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/51ff868300679fb68a5bed9e75d244e59b598e40